assets/terraform-modules/worker/main.tf

```diff
@@ -37,10 +37,8 @@ data "ct_config" "config" {
     cluster_domain_suffix     = var.cluster_domain_suffix
     host_dns_ip               = var.host_dns_ip
     kubelet_docker_extra_args = []
-    # Here we set default labels for worker nodes.
-    kubelet_labels = length(var.kubelet_labels) > 0 ? var.kubelet_labels : {
-      "node.kubernetes.io/node" = ""
-    }
+    # Here we set default labels for worker nodes and append with user provided labels.
+    kubelet_labels = merge({ "node.kubernetes.io/node" = "" }, var.kubelet_labels)
```

Comment:
I don't think this patch is correct. If as a user I don't want to have `"node.kubernetes.io/node"` label, as I consider it to be  a clutter.

I think we should modify the test code to select on `node.kubernetes.io/controller!=true`, rather than on `node.kubernetes.io/node=`, as `node.kubernetes.io/controller` is part of the "spec" and has stronger meaning than `node.kubernetes.io/node` (which right now is optional).

Reply:
done.